.gitignore

```diff
@@ -3,6 +3,7 @@
 
 .vim
 .env
+.venv
```

Comment:
No. That's a user thing it does not belong here.

Please use a global `.gitignore` for you (then you also only define it once on your machine).